08-Image-coords/Image_coordinates_tutorial.ipynb

```diff
@@ -0,0 +1,257 @@
+    "%matplotlib inline\n",
+    "from matplotlib import pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astropy.utils.data import get_pkg_data_filename\n",
+    "from astropy.io import fits\n",
+    "from astropy import wcs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating a WCS object from the header of a FITS file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Open a FITS image with `astropy.fits` and examine the contents."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_file = fits.open('data/w5.fits')\n",
+    "\n",
+    "image_file.info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To create a WCS object pass the header with the WCS keywords to astropy.wcs.WCS. In this case it is the primary header. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w = wcs.WCS(image_file[0].header)\n",
+    "w"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For this case, `fits.getheader` may be used to save a step."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w = wcs.WCS(fits.getheader('data/w5.fits'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Check the physical type of each axis"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w.world_axis_physical_types"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Print the number of pixel and world axes, and the shape of the data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(w.pixel_n_dim, w.world_n_dim, w.array_shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Transforming between pixel coordinates and sky coordinates"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To perform the transformation from detector to sky, pass x-pixel and y-pixel values to the `pixel_to_world` method.\n",
+    "\n",
+    "The inputs can be numbers, numpy arrays or array-like objects. See the [Pixel Conventions and Definitions](https://docs.astropy.org/en/stable/wcs/index.html#pixel-conventions) for an explanation of the pixel numbering convention."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "coord = w.pixel_to_world([10, 100], [24, 500.5])\n",
```

Comment:
does `pixel_to_world` return a SkyCoord object? If so, it would be good to explicitly state that in the description.